src/print.rs

```diff
@@ -41,16 +41,19 @@ pub fn prompt(args: ArgMatches) {
         .flatten()
         .collect::<Vec<Module>>(); // Remove segments set to `None`
 
+    let mut line_break = true;
```

Comment:
I think this may be a better variable name, since it's not only to do with the `line_break` module.

```suggestion
	// Skip printing the prefix of the first module
    let mut print_without_prefix = true;
```

Reply:
very good point. updated!